app/mailboxes/support_mailbox.rb

```diff
@@ -41,6 +41,10 @@ def decorate_mail
     @processed_mail = MailPresenter.new(mail, @account)
   end
 
+  def original_sender
```

Comment:
let's have this method in the `MailPresenter` itself so that all custom logic over the mail object remains in the same place.

Reply:
Cool 💯